app/src/main/res/values/strings.xml

```diff
@@ -653,4 +653,8 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="label_housenumber">house number</string>
     <string name="label_blocknumber">block number</string>
     <string name="quest_busStopShelter_covered">Whole stop is covered</string>
+    <string name="quest_ferry_pedestrian_name_title">Does %s transport pedestrians?</string>
+    <string name="quest_ferry_pedestrian_title">Does transport pedestrians?</string>
```

Comment:
Is not there a "…" or so missing? Grammatically the subject is missing here.

Reply:
Or possibly "Does this transport…"?

Reply:
You are late to the party. This is long solved.